guide/14-deep-learning/point_cloud_classification_using_point_transformer.ipynb

```diff
@@ -0,0 +1,458 @@
+{
```

Comment:
<ul><li>change the text in the links to [1] from <em>References</em> to <em>references. </em>The upper case R does not lead to correct navigation.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>